A list of low hanging docs changes from our feature testing (#426)
infrawatch/documentation: doc-Service-Telemetry-Framework/modules/proc_creating-an-alert-route-with-templating-in-alertmanager.adoc

$ oc project service-telemetry
----

> . Create the necesarry alermanager config in a file called alertmanager.yaml, for example:

Reviewer: ```suggestion
. Create the necessary alertmanager config in a file called alertmanager.yaml, for example:
```

+
[source,yaml]
----

> $ cat alertmanager.yaml

Reviewer: I feel like you want to follow some of the patterns like `cat > alertmanager.yaml <<EOF` here.

$ oc project service-telemetry
----

> . Create the necessary alertmanager config in a file called alertmanager.yaml, for example:

Reviewer: ```suggestion
. Create the necessary Alertmanager config in a file called alertmanager.yaml, for example:
```

Reviewer: We referred to it earlier with a capital, so just working through for consistency.

+
[source,yaml]
----
$ cat > alertmanager.yaml <<EOF
global:
  resolve_timeout: 10m
  slack_api_url: <slack_api_url>
receivers:
  - name: slack
    slack_configs:
    - channel: #stf-alerts
      title: |-
        ...
      text: >-
        ...
route:
  group_by: ['job']
  group_wait: 30s
  group_interval: 5m
  repeat_interval: 12h
  receiver: 'slack'
EOF
----
. Generate the config manifest and add it to the `ServiceTelemetry` object for your {ProjectShort} deployment:
+
[source,bash,options="nowrap"]
----
$ CONFIG_MANIFEST=$(oc create secret --dry-run=client generic alertmanager-default --from-file=alertmanager.yaml -o json)
$ oc patch stf default --type=merge -p '{"spec":{"alertmanagerConfigManifest":'"$CONFIG_MANIFEST"'}}'
----
. Verify that the configuration has been applied to the secret:
+
[NOTE]

> There will be a short delay as the operators update each object

Reviewer: ```suggestion
There will be a short delay as the operators update each object.
```
